codemirror/codemirror.d.ts

```diff
@@ -1125,6 +1125,127 @@ declare namespace CodeMirror {
         severity?: string;
         to?: Position;
     }
+
+    /**
+     * A function that calculates either a two-way or three-way merge between different sets of content.
+     */
+    function MergeView(element: HTMLElement, options?: MergeViewEditorConfiguration): MergeViewEditor;
+
+    /**
+     * Options available to MergeView.
+     */
+    interface MergeViewEditorConfiguration extends EditorConfiguration {
```

Comment:
Can we put interfaces below into `namespace MergeView`?

This adds quite a lot of minute noisy names into the existing namespace.